README.md

```diff
@@ -1,6 +1,6 @@
 # Data repository for covid19japan.com
 
-[data.covid19japan.com](https://data.covid19japan.com)
+[covid19japan.com/](https://covid19japan.com/)
```

Comment:
We might want to clarify that it is for this primary site, but hosted on the subdomain?